src/creature.cpp

```diff
@@ -898,6 +899,16 @@ void Creature::deal_damage_handle_type( const damage_unit &du, bodypart_id bp, i
             div = 3.0f;
             break;
 
+        case DT_CUT:
+        case DT_STAB:
+        case DT_BULLET:
+            // these are bleed inducing damage types
+            if( is_player() || is_npc() ) {
```

Comment:
is there really no is_character() function to use here? and is_player() will be on its way out so i'd rather it be is_avatar() || is_npc() if not.